src/main/resources/swagger/api-docs.yaml

```diff
@@ -6811,15 +6843,9 @@ definitions:
       message:
         type: string
         description: extended explanation for the import job.
-      timestamp:
-        type: integer
-        format: int64
-        description: time of the response
     required:
       - jobId
       - status
-      - message
-      - timestamp
```

Comment:
message is optional; timestamp was never part of the payload. These changes bring the swagger def in line with reality; I am not changing any behavior here.